### PR TITLE
Improve close behavior some.

### PR DESCRIPTION
**Pull request by brendandburns**
Possible fixes for #687 

@ceeaspb

### CODE REVIEW

File: util/src/main/java/io/kubernetes/client/util/WebSocketStreamHandler.java [hunk 1]
```diff
@@ -86,6 +86,7 @@ public void textMessage(Reader in) {
   protected void handleMessage(int stream, InputStream inStream) throws IOException {
     OutputStream out = getSocketInputOutputStream(stream);
     ByteStreams.copy(inStream, out);
+    inStream.close();
```
Comment (yue9944882) [inline util/src/main/java/io/kubernetes/client/util/WebSocketStreamHandler.java]: nit: how about closing the stream in a try-finally block?

Comment (brendandburns) [reply util/src/main/java/io/kubernetes/client/util/WebSocketStreamHandler.java]: done.

**Comment by brendandburns** [comment]
Comment addressed please take another look.

**Comment by yue9944882** [comment]
> [ERROR] To fix formatting errors, run "mvn fmt:format"
[ERROR] Non complying file: /home/travis/build/kubernetes-client/java/util/src/main/java/io/kubernetes/client/util/WebSocketStreamHandler.java

ci is failing on formatting error

**Comment by brendandburns** [comment]
CI is fixed. THanks!

**Pull request merged**